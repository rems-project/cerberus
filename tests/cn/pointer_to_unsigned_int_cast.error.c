void f() {
  int x=1;
  (unsigned int)&x;
  return;
}

int main(void) {
  f();
  return 0;
}