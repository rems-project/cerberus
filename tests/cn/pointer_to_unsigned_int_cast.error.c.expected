return code: 1
[1/2]: f -- fail
[2/2]: main -- pass
tests/cn/pointer_to_unsigned_int_cast.error.c:3:3: error: integer value not representable at type unsigned int
  (unsigned int)&x;
  ^~~~~~~~~~~~~~~~ 
Value: &x
State file: file:///tmp/state__pointer_to_unsigned_int_cast.error.c__f.html
